MAINTAINERS.md

```diff
@@ -25,6 +25,7 @@ Code and Systems maintainers
 - [Lucas Rodriguez Benitez](https://github.com/LucasRodriguez)
 - [Anais Ofranc](https://github.com/Consianimis)
 - [Hamza Tokuchi](https://github.com/Meuko) - [BlockLab](https://www.blocklab.nl/)
+- [Biswashree Dey](https://github.com/biscuitdey)
```

Comment:
this list is not updated, lets not add to it, probably should be removed completely?

Reply:
Done.